src/main/java/it/gov/pagopa/pu/workflow/wf/debtposition/custom/fine/DebtPositionFineClientImpl.java

```diff
@@ -22,7 +21,7 @@ public DebtPositionFineClientImpl(WorkflowService workflowService) {
   }
 
   @Override
-  public String handleFineReductionExpiration(Long debtPositionId, PaymentEventRequestDTO paymentEventRequestDTO, FineWfExecutionConfig executionParams) {
+  public String expireFineReduction(Long debtPositionId, FineWfExecutionConfig executionParams) {
```

Comment:
better to call it wfExecutionConfig
```suggestion
  public String expireFineReduction(Long debtPositionId, FineWfExecutionConfig wfExecutionConfig) {
```